orchestration/dap_orchestration/solids.py

```diff
@@ -21,47 +22,52 @@ def extract_records(context: AbstractComputeExecutionContext) -> None:
         "pullDataDictionaries": "true" if context.solid_config["pull_data_dictionaries"] else "false",
         "outputPrefix": f"{context.resources.refresh_directory}/{context.solid_config['output_prefix']}",
         "endTime": context.solid_config["end_time"],
-        "target_class": context.solid_config["target_class"],
-        "apiToken": context.solid_config["api_token"],
-        "scala_project": "dog-aging-hles-extraction"
+        "apiToken": context.solid_config["api_token"]
     }
-    context.resources.beam_runner.run(arg_dict)
+    context.resources.beam_runner.run(arg_dict,
+                                      target_class=context.solid_config["target_class"],
+                                      scala_project=context.solid_config["scala_project"])
 
 
-def _build_extract_config(config: dict[str, str], target_class: str, output_prefix: str) -> dict[str, str]:
+def _build_extract_config(config: dict[str, str], output_prefix: str,
+                          target_class: str, scala_project: str) -> dict[str, str]:
     return {
         "pull_data_dictionaries": config["pull_data_dictionaries"],
         "output_prefix": output_prefix,
         "end_time": config["end_time"],
         "api_token": config["api_token"],
-        "target_class": target_class
+        "target_class": target_class,
+        "scala_project": scala_project,
     }
 
 
 @configured(extract_records)
 def hles_extract_records(config: dict[str, str]) -> dict[str, str]:
     return _build_extract_config(
-        config,
-        "org.broadinstitute.monster.dap.hles.HLESurveyExtractionPipeline",
-        "raw"
+        config=config,
+        output_prefix="raw",
+        target_class="org.broadinstitute.monster.dap.hles.HLESurveyExtractionPipeline",
+        scala_project="dog-aging-hles-extraction"
```

Comment:
small nit but if the scala project is the same for all these, we can pull it into a `project` variable so that if we ever have to change it we only need to change it in one place

Reply:
appreciate all the nits!